### DISCUSSION TITLE
Fortify support

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [filamentphp/filament#4467] [Help] [answered]
Does Filament supports Laravel fortify?

**Comment by danharrin** [comment, marked as answer]
In what way? The admin panel doesnt dictate your frontend choices

**Comment by madalinignisca** [comment]
Probably some people that look at fortify are wondering about 2FA. I know there's a plugin, problematic for 3.x, which has 2FA, but probably 2FA should be part of Filament's core features for authentication.
